src/gradleperfsmoketest.sh

```diff
@@ -0,0 +1,22 @@
+#!/bin/bash
+
+gradleperfsmoketest () {
+
+  if [[ -n "${google_project}" ]] && [[ "${google_application_credentials}" != "" ]] && [ -f ${google_application_credentials} ]; then
+    export GOOGLE_APPLICATION_CREDENTIALS=${google_application_credentials}
+    export GOOGLE_CLOUD_PROJECT=${google_project}
+    export GOOGLE_CLOUD_DATA_PROJECT=${google_data_project}
+
+    cd ${GITHUB_WORKSPACE}/${workingDir}/datarepo-client
+    ../gradlew clean assemble
+    cd ${GITHUB_WORKSPACE}/${workingDir}
+    export ORG_GRADLE_PROJECT_datarepoclientjar=.$(ls ./datarepo-client/build/libs/*jar)
```

Comment:
I'm not familiar with the .$ syntax, so ignore this if it doesn't make sense.

I think this path is going to be relative to workingDir right? Does gradle still find the JAR file after you cd into the datarepo-clienttests below?

Reply:
This is the output: ORG_GRADLE_PROJECT_datarepoclientjar=../datarepo-client/build/libs/datarepo-client-1.0.74-SNAPSHOT.jar
Without adding that extra '.' at the front, it wouldn't change directory correctly. I feel pretty confident that it works.  But, I'm totally open to other options of getting this path.

Reply:
Oh okay I understand what it's doing now, thanks for clarifying. No need to change anything, since this already works, but you could also set the path after you change directories:
cd ${GITHUB_WORKSPACE}/${workingDir}/datarepo-clienttests 
export ORG_GRADLE_PROJECT_datarepoclientjar=$(ls ../datarepo-client/build/libs/*jar)

Only difference is moving the dot inside, I just got tripped up on the .$ piece -- sorry for the distraction!

Reply:
That is much cleaner - I'll do that!